cve_bin_tool/checkers/openssh.py

```diff
@@ -16,7 +17,7 @@ def get_version(lines, filename):
 
     VPkg: openbsd, openssh
     """
-    regex = re.compile(r"OpenSSH_([0-9]+\.[0-9]+[0-9a-z\s]*)")
+    regex = [r"openssh: ([0-9]+\.[0-9]+)"]
```

Comment:
In general, we don't want to remove signatures unless they're bad (and this one isn't as far as I can tell).  You can add if you find out there are things that the old one misses, but that's why the regex list is a list -- you can have multiple things in it.

Reply:
I have added the older signature along with the new one.